src/oneD/IonFlow.cpp

```diff
@@ -0,0 +1,416 @@
+//! @file IonFlow.cpp
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at http://www.cantera.org/license.txt for license and copyright information.
+
+#include "cantera/oneD/IonFlow.h"
+#include "cantera/oneD/StFlow.h"
+#include "cantera/base/ctml.h"
+#include "cantera/transport/TransportBase.h"
+#include "cantera/numerics/funcs.h"
+
+using namespace std;
+
+namespace Cantera
+{
+
+IonFlow::IonFlow(IdealGasPhase* ph, size_t nsp, size_t points) :
+    FreeFlame(ph, nsp, points),
+    m_phase(1),
+    m_inletVoltage(0),
+    m_outletVoltage(0),
+    m_kElectron(Undef)
```

Comment:
You should use `npos` (which is of type `size_t`) instead of `Undef` (which is a `double`) for this purpose.